homeassistant/components/alexa/capabilities.py

```diff
@@ -1854,3 +1863,44 @@ def configuration(self):
         When false, Alexa does not send the Resume directive.
         """
         return {"allowRemoteResume": self._allow_remote_resume}
+
+
+class AlexaCameraStreamController(AlexaCapability):
+    """Implements Alexa.CameraStreamController.
+
+    https://developer.amazon.com/docs/device-apis/alexa-camerastreamcontroller.html
+    """
+
+    supported_locales = {
+        "de-DE",
+        "en-AU",
+        "en-CA",
+        "en-GB",
+        "en-IN",
+        "en-US",
+        "es-ES",
+        "fr-FR",
+        "it-IT",
+        "ja-JP",
+    }
+
+    def name(self):
+        """Return the Alexa API name of this interface."""
+        return "Alexa.CameraStreamController"
+
+    def camera_stream_configurations(self):
+        """Return cameraStreamConfigurations object."""
+        camera_stream_configurations = [
+            {
+                "protocols": ["HLS"],
+                "resolutions": [
+                    {"width": 1920, "height": 1080},
+                    {"width": 1280, "height": 720},
+                    {"width": 640, "height": 480},
```

Comment:
We don't currently know the configurations of our streams, should we just return 1 entry here ?

Reply:
This is something I'm currently testing against the Alexa API to see if I can remove, or set to one resolution. I originally included them to cover all Alexa devices, but since the API doesn't really care what video I send, regardless of the request, it may need be needed.